### PR TITLE
Cleanup readme

### PR DESCRIPTION
**Pull request by suhasgaddam**
Fix scalajs badge.

The scalajs badge version needs to be fixed in sbt-org-policies.

The available badges: http://www.scala-js.org/assets/badges/

`0.6.20 is also forward binary compatible with 0.6.{17-19}` from https://www.scala-js.org/news/2017/09/01/announcing-scalajs-0.6.20/

### CODE REVIEW

**Comment by andyscott** [comment]
Do you know if we need to update the original readme file in the modules/readme directory?

**Comment by suhasgaddam** [comment]
Interesting, I didn't notice that iota has a special `readme` project and `tutReadme`.

If I am reading and understanding the order of things, `tutReadme` will run in `orgScriptCI` and during an `sbt release`(which occurs in `orgAfterCISuccess`) the root `README.md` will get its badges updated/added.

**Comment by andyscott** [comment]
Yep. The readme has code that gets compiled and type checked. It's pretty easy to set up a separate module for this, albeit a bit confusing when someone wants to quickly make README updates!

File: modules/readme/src/main/tut/README.md [hunk 1]
```diff
@@ -1,9 +1,3 @@
-[comment]: # (Start Badges)
```
Comment (andyscott) [inline modules/readme/src/main/tut/README.md]: @juanpedromoreno can you weigh in on this and verify if it's the correct to remove these markers?

Comment (juanpedromoreno) [reply modules/readme/src/main/tut/README.md]: Yeah, as far as I remember, the plugin will perform the badge updates the root's README. So I guess we might not need those markers there.

**Comment by juanpedromoreno** [comment]
@suhasgaddam you might need to set `orgSupportedScalaJSVersion := Some("0.6.17")` to establish the latest scalajs available badge.

File: modules/readme/src/main/tut/README.md (hunk 1 above)
Comment (andyscott) [reply modules/readme/src/main/tut/README.md]: I think you mentioned before that we needed an empty newline at the beginning of the file? Is that still the case?

**Pull request merged**